platforms/database.go

```diff
@@ -72,6 +72,15 @@ func normalizeOS(os string) string {
 	return os
 }
 
+// normalizeArch normalizes the architecture.
```

Comment:
copy paste error.

Are features being used? I didn't think this was used anywhere